ipyleaflet/leaflet.py

```diff
@@ -798,6 +798,125 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class ImageService(Layer):
+    """ImageService class
+
+    Image Service layer for raster data served through a web service
+
+    Attributes
+    ----------
+    url: string, default ""
+        URL to the image service
+    f: string, default "image"
+        response format (use `'image'` to stream as bytes)
+    format: string, default "jpgpng"
+        format of exported image
+
+        - ``jpgpng``
```

Comment:
Did you mean:
```suggestion
        - `'jpgpng'`
```

Reply:
put the docstring edits in! thanks @davidbrochart

Reply:
But shouldn't it be:
```
`'jpgpng'`
```
instead of:
```
``"jpgpng"``
```

Reply:
are the docstrings in ipyleaflet rendered from markdown and not restructuredtext?

Reply:
Yes you're right, I think it's reStructuredText :+1: